.yamato/upm-ci.yml

```diff
@@ -1,3 +1,23 @@
+release_4_2_nightly_trigger_editors:
```

Comment:
Can you add a comment (line that starts with '#') to describe what we should do when we release another version, when to update these, etc?

In particular I'm wondering why we have a mix of 2022.2 and 2022.1

Reply:
for the mix, I followed the versions from Marie's comment here: https://github.com/Unity-Technologies/com.unity.formats.fbx/pull/630#issuecomment-1124703236

I think it is the latest version that was available when that version was released